docs/connector.rst

```diff
@@ -73,6 +73,16 @@ Once you have created a ``Connector`` instance, :meth:`Connector.get_output()`
 returns the :class:`Output` that allows writing data, and :meth:`Connector.get_input()`
 returns the :class:`Input` that allows reading data.
 
+.. note::
+
+  If the *domain_participant* you load contains both *data_writers* (Outputs) and
+  *data_readers* (Inputs) for the same topics, when you write data, the Inputs
+  will receive the data even before you call :meth:`Connector.get_input()`.
+  To avoid that, you can configure the *subscriber* that contains the
+  *data_reader* with ``<subscriber_qos>/<entity_factory>/<autoenable_created_entities>``
+  set to ``false``. In that case, the Inputs will only receive data after you
+  call :meth:`Connector.get_input()`.
+
```

Comment:
Do you think we should mention here the bigger issue of the un-wanted data_reader's queue filling up and blocking the writer in reliable communication?

Reply:
I didn't want to get into everything that can go wrong. That's the most obvious side effect, but there could be more. I think I'll leave it to the reader to figure it out.

Reply:
The human reading the documentation, that is.